lib/index.ts

```diff
@@ -93,13 +95,15 @@ function buildDepTreeFromFiles(
   targetFramework?: string,
 ) {
   if (!root || !manifestFilePath) {
-    throw new Error('Missing required parameters for buildDepTreeFromFiles()');
+    throw new OpenSourceEcosystems.MissingPayloadError(
+      'Missing required parameters for building dependency tree from files',
+    );
   }
 
   const manifestFileFullPath = path.resolve(root, manifestFilePath);
 
   if (!fs.existsSync(manifestFileFullPath)) {
-    throw new Error(
+    throw new OpenSourceEcosystems.CannotGetFileFromSourceError(
       'No packages.config, project.json or project file found at ' +
         `location: ${manifestFileFullPath}`,
     );
```

Comment:
Can you add `location` to `extras` instead of in the details? It's easier to search for it in DataDog that way.